### PR TITLE
Add Open Images format

### PR DESCRIPTION
**Pull request by kirill-sizov**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [x] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [x] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [x] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by zhiltsov-max — APPROVED** [review]

**Review by zhiltsov-max — COMMENTED** [review]
Review the implementation wrt #3790

**Review by zhiltsov-max — APPROVED** [review]

File: cvat/apps/dataset_manager/formats/openimages.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    )
+
+    for pattern in image_desc_patterns:
+        for path in glob.glob(pattern):
+            with open(path, 'r') as desc:
+                next(desc)
+                for row in desc:
+                    yield row.split(',')[0]
+
+@exporter(name='Open Images', ext='ZIP', version='1.0')
```
Comment (zhiltsov-max) [inline cvat/apps/dataset_manager/formats/openimages.py:39]: Maybe, we should be more specific about the version and add `v6`.

**Review by zhiltsov-max — DISMISSED** [review]

**Comment by nmanovic** [comment]
@kirill-sizov , image meta information is provided only for annotatated images. Is it expected?
I have approximately 200 images in my task. I have annotated 9 first frames.


```bash
$ cat images.meta
# <image name> <height> <width>

Abyssinian_123 333 500
Abyssinian_71 539 833
Bengal_197 206 300
Birman_128 375 500
Birman_43 500 340
Birman_44 375 500
Birman_82 324 448
Bombay_104 250 167
Bombay_157 849 704
```

**Comment by kirill-sizov** [comment]
> @kirill-sizov , image meta information is provided only for annotatated images. Is it expected? I have approximately 200 images in my task. I have annotated 9 first frames.
> 
> ```shell
> $ cat images.meta
> # <image name> <height> <width>
> 
> Abyssinian_123 333 500
> Abyssinian_71 539 833
> Bengal_197 206 300
> Birman_128 375 500
> Birman_43 500 340
> Birman_44 375 500
> Birman_82 324 448
> Bombay_104 250 167
> Bombay_157 849 704
> ```

@nmanovic, Yes it's expected, because we need information about image size only for images that has annotation objects.

**Comment by nmanovic** [comment]
> > @kirill-sizov , image meta information is provided only for annotatated images. Is it expected? I have approximately 200 images in my task. I have annotated 9 first frames.
> > ```shell
> > $ cat images.meta
> > # <image name> <height> <width>
> > 
> > Abyssinian_123 333 500
> > Abyssinian_71 539 833
> > Bengal_197 206 300
> > Birman_128 375 500
> > Birman_43 500 340
> > Birman_44 375 500
> > Birman_82 324 448
> > Bombay_104 250 167
> > Bombay_157 849 704
> > ```
> 
> @nmanovic, Yes it's expected, because we need information about image size only for images that have annotation objects.

I'm not sure I understand "we". Cityscapes dataset can be used by multiple scripts around the world which can rely on the file. What is the answer from the dataset format point of view?

**Comment by kirill-sizov** [comment]
> > > @kirill-sizov , image meta information is provided only for annotatated images. Is it expected? I have approximately 200 images in my task. I have annotated 9 first frames.
> > > ```shell
> > > $ cat images.meta
> > > # <image name> <height> <width>
> > > 
> > > Abyssinian_123 333 500
> > > Abyssinian_71 539 833
> > > Bengal_197 206 300
> > > Birman_128 375 500
> > > Birman_43 500 340
> > > Birman_44 375 500
> > > Birman_82 324 448
> > > Bombay_104 250 167
> > > Bombay_157 849 704
> > > ```
> > 
> > 
> > @nmanovic, Yes it's expected, because we need information about image size only for images that have annotation objects.
> 
> I'm not sure I understand "we". Cityscapes dataset can be used by multiple scripts around the world which can rely on the file. What is the answer from the dataset format point of view?

The file `image.meta` is generated by Datumaro to read the dataset more efficiently. The format stores annotations in a normalized form relative to the size of the image. To avoid reading images, Datumaro uses a workaround in the form of this meta file. Original dataset doesn't contain such information.

File: cvat/apps/dataset_manager/formats/openimages.py (hunk 1 above)
Comment (kirill-sizov) [reply cvat/apps/dataset_manager/formats/openimages.py]: Done.

**Review by nmanovic — APPROVED** [review]

**Pull request merged**